core/src/test/java/org/elasticsearch/search/aggregations/metrics/percentiles/InternalPercentilesRanksTestCase.java

```diff
@@ -64,5 +75,36 @@ protected final void assertFromXContent(T aggregation, ParsedAggregation parsedA
         assertTrue(parsedClass.isInstance(parsedAggregation));
     }
 
+    public void testPercentilesRanksIterators() throws IOException {
```

Comment:
I addressed this in #24183 where I added a parse() method that accepts a `shuffled` boolean.